### ISSUE TITLE
Improve accessability support

### ISSUE THREAD

**Issue by anoQlik** [qlik-oss/nebula.js#607]
1. <svg> tags are decorative and provide no information to users should have aria-hidden="true" attributes (applies to filterpane) 

2. Buttons that are not marked up as such; screen readers will not identify them as actionable and they will not be usable by keyboard users.
elements:
"Rating Class" > more button (expanded) > list of buttons
Applied Filter button "Round(rating)" for example

3. Expanded content uses ARIA menu roles, which are only intended for desktop-style application menus. Improper use of these roles often prevents screen reader users from accessing the expanded content.
elements:
"Rating Class" > more button (expanded) > list of buttons
Remove the ARIA menu roles from this menu, including role="menu", role="menuitem", and role="menubar"

4. Focus indicator missing
"Rating Class" > more button (expanded) > list of buttons
A focus indicator should always be visible, and have sufficient contrast. It is best to intentionally style the focus indicator for all links, buttons, fields, etc. We encourage you to change the shape somehow (add a border, box, underline, etc.) as well as choose a color that has sufficient contrast, and test each element with the tab key to make sure a focus indicator is always visible for each focusable element.

5. Elements control expandable content, but it is not indicated by screen readers.
"Rating Class" > more button
Applied filter button
Add an aria-controls attribute to the elements that control the expandable content and set the values of those attributes to equal the ids of the tags that wrap the expanded content. Also add an aria-expanded="true" attribute to those same control elements when content is expanded, and aria-expanded="false" otherwise.

6. Ensure that information of the selectable nature of elements is conveyed.

7. Buttons incorrectly labeled
Add screen reader only text or an aria-label attribute to each of these buttons which tells users where they go and/or what they do. These are labeled with title attributes, but this is not sufficient for these labels to be ready consistently across various browser and screen reader combinations.

8. Form field for search in listbox should be correctly labeled
